src/dockleHelper.ts

```diff
@@ -92,6 +92,23 @@ export function getText(dockleStatus: number): string {
     return `**Best Practices Violations** -${clusteredViolations ? clusteredViolations : '\nNone found.'}`;
 }
 
+export function getDockleFilteredOutput(): any {

```

Comment:
Suggestion: `getFilteredOutput`. Or `getScanOutput` may be a better name?

Reply:
changed to getFilteredOutput